### PR TITLE
Merge xcode10.1 support into 16.0 P1

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by spouliot** [comment]
xtro random issue filed -> https://github.com/xamarin/maccore/issues/1111
watchOS known issue HE0038 -> https://github.com/xamarin/maccore/issues/581

File: src/GameController/GCExtendedGamepadSnapshot.cs [hunk 1]
```diff
@@ -47,6 +47,12 @@ public struct GCExtendedGamepadSnapShotDataV100 {
 		public float /* float_t = float */ LeftTrigger;
 		public float /* float_t = float */ RightTrigger;
 
+		// radar: https://trello.com/c/7FoGTORD (GCExtendedGamepadSnapShotDataV100 struct size / alignment not backward compatible)
```
Comment (VincentDondain) [inline src/GameController/GCExtendedGamepadSnapshot.cs]: Minor but that can be deleted now.

Comment (spouliot) [reply src/GameController/GCExtendedGamepadSnapshot.cs]: yes, but it would be more useful in master than in a throw-away branch :)

**Comment by spouliot** [comment]
@VincentDondain 16.0 P1 has not been released yet - so no version bump is needed

**Pull request merged**